rules/sonic-ycable.dep

```diff
@@ -0,0 +1,16 @@
+SPATH       := $($(SONIC_YCABLE_PY2)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/sonic-ycable.mk rules/sonic-xcvrd.dep
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+SMDEP_FILES := $(addprefix $(SPATH)/,$(shell cd $(SPATH) && git ls-files))
+
+$(SONIC_YCABLE_PY2)_CACHE_MODE  := GIT_CONTENT_SHA
+$(SONIC_YCABLE_PY2)_DEP_FLAGS   := $(SONIC_COMMON_FLAGS_LIST)
+$(SONIC_YCABLE_PY2)_DEP_FILES   := $(DEP_FILES)
+$(SONIC_YCABLE_PY2)_SMDEP_FILES := $(SMDEP_FILES)
+$(SONIC_YCABLE_PY2)_SMDEP_PATHS := $(SPATH)
```

Comment:
We are moving to PY3 only, then why PY2 support?

Reply:
fixed, removed PY2